README.md

```diff
@@ -208,6 +208,12 @@ Allows custom monitoring events to be added into the Node Application Metrics ag
 * `type` (String) the name you wish to use for the data. A subsequent event of that type will be raised, allowing callbacks to be registered for it.
 * `data` (Object) the data to be made available with the event. The object must not contain circular references, and by convention should contain a `time` value representing the milliseconds when the event occurred.
 
+### appmetrics.writeHeapSnapshot([filename],[callback])
+Dumps the v8 heap via `heapdump`. For more information, see [the heapdump README](https://github.com/bnoordhuis/node-heapdump/blob/master/README.md)
+
+### appmetrics.triggerNodeReport()
```

Comment:
Not a good point. Are you saying nodereport is a singleton? That only one instance of it can be required at any time, and it will somehow break if required multiple times, or if multiple instances of it are required into the same application? Thats not a well behaved npm package, and should be fixed in nodereport.

Requiring nodereport should have zero-side effects, and does (to my knowledge), so I don't see why requiring it from appmetrics should be an issue. It would be possible to lazy-require it by defining `appmetrics.nodereport` as a property getter.